scripts/travis-test.sh

```diff
@@ -18,17 +18,19 @@
 set -o nounset
 set -o errexit
 
+RTN=0
 SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd -P)"
 source "${SCRIPT_DIR}/travis-common.sh"
 
 log_and_run() {
   echo $*
-  exec $*
+  (exec $*) && true
```

Comment:
Thanks! I think the whole line is equivalent to just `$*`, but if it works with Travis, it's probably best not to fiddle with it :-)

The new code looks correct to me and appears to fix the bug, so this was mere curiosity.